pypyr-ops/Dockerfile

```diff
@@ -6,20 +6,38 @@
 # build me like this from repo root:
 # docker build -t pypyr/pypyr-ops -t pypyr/pypyr-ops:latest -t pypyr/pypyr-ops:1.0.0 -f pypyr-ops/Dockerfile .
 
-FROM pypyr/pypyr-go:pypyr1.1.1-go1.11.1
+FROM pypyr/pypyr-go:pypyr1.2.1-go1.11.2
 
-ARG terraformversion=0.11.8
+ARG terraformversion=0.11.10
 
-ARG awsversion=1.16.37
+ARG awsversion=1.16.48
 
-RUN  apt-get update && apt-get install -y apt-transport-https; \
-        wget https://releases.hashicorp.com/terraform/${terraformversion}/terraform_${terraformversion}_linux_amd64.zip; \
-         curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -; \
-         echo "deb http://apt.kubernetes.io/ kubernetes-xenial main" | tee -a /etc/apt/sources.list.d/kubernetes.list; \
+RUN  apt-get update && apt-get install -y apt-transport-https \
+        ca-certificates \
+        curl \
+        gnupg2 \
+        software-properties-common && \
+        wget https://releases.hashicorp.com/terraform/${terraformversion}/terraform_${terraformversion}_linux_amd64.zip && \
+         curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add - && \
+         echo "deb http://apt.kubernetes.io/ kubernetes-xenial main" | tee -a /etc/apt/sources.list.d/kubernetes.list && \
+         curl -fsSL https://download.docker.com/linux/$(. /etc/os-release; echo "$ID")/gpg > /tmp/dkey && apt-key add /tmp/dkey && \
+         add-apt-repository \
+            "deb [arch=amd64] https://download.docker.com/linux/$(. /etc/os-release; echo "$ID") \
+            $(lsb_release -cs) \
+            stable" && \
          apt-get update && apt-get install -y unzip \
-         kubectl; \
-         unzip terraform_${terraformversion}_linux_amd64.zip; \
-         mv terraform /usr/local/bin; \
-         rm -f terraform_${terraformversion}_linux_amd64.zip; \
-         pip install awscli==${awsversion}; \
+         kubectl \
```

Comment:
you can install kubectl with a more specific version like this:
```
curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.12.0/bin/linux/amd64/kubectl
chmod +x ./kubectl
mv ./kubectl /usr/local/bin/kubectl
```
you can then parameterize/ARG the version number. . .